### PR TITLE
jk -- Add GetSystemInfo (NETSDK-41) and VerifySystemHealth (NETSDK-40…

### PR DESCRIPTION
**Pull request by berkeleysquare**
…) calls

### CODE REVIEW

File: Ds3/Calls/GetSystemInformatoinResponse.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ *  </BuildInformation>
+ *  <SerialNumber>UNKNOWN</SerialNumber>
+ * </Data>
+ */
+
+namespace Ds3.Calls
+{
+    public class GetSystemInformationResponse
+    {
+        public string ApiMC5Major { get; set; }
```
Comment (rpmoore) [inline Ds3/Calls/GetSystemInformatoinResponse.cs]: Should these setters be private?

File: Ds3/ResponseParsers/GetSystemInformationParser.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                    var buildParams = root.Element("BuildInformation");
+                    string BuildBranch = buildParams.TextOf("Branch");
+                    string BuildRev = buildParams.TextOf("Revision");
+                    string BuildVersion = buildParams.TextOf("Version");
+                    string ApiVersion = root.TextOf("ApiVersion");
+                    string MC5Major = string.Empty;
+                    string MC5Full = string.Empty;
+                    if (!string.IsNullOrEmpty(ApiVersion))
+                    {
+                        string[] mc5s = ApiVersion.Split('.');
```
Comment (rpmoore) [inline Ds3/ResponseParsers/GetSystemInformationParser.cs]: I don't know if this is needed.  We've not done it in the other SDKs.

Comment (berkeleysquare) [inline Ds3/ResponseParsers/GetSystemInformationParser.cs]: You’re referring to the split of API version into the two MC5’s – or the whole parser?

jk

Comment (rpmoore) [inline Ds3/ResponseParsers/GetSystemInformationParser.cs]: The splitting of it into two variables.

File: Ds3/Calls/GetSystemInformatoinResponse.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using System.Net;
+
+using Ds3.Models;
+using Ds3.Runtime;
+
+/* <Data>
```
Comment (rpmoore) [inline Ds3/Calls/GetSystemInformatoinResponse.cs]: Do we need to have the xml sample here?

File: Ds3/ResponseParsers/GetSystemInformationParser.cs [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ */
+
+using System;
+using System.IO;
+using System.Net;
+using Ds3.Calls;
+using Ds3.Models;
+using Ds3.Runtime;
+
+/* <Data>
```
Comment (rpmoore) [inline Ds3/ResponseParsers/GetSystemInformationParser.cs]: Same comment about needing this xml sample

**Pull request merged**